### PR TITLE
i18n(zh-cn): Update `markdoc.mdx`

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update `markdoc.mdx`

#### Related issues & labels (optional)

- #7743 
- #7744

### CODE REVIEW

**Comment by yanthomasdev** [comment]
Heads up we got a [few more updates on this page](https://github.com/withastro/docs/commit/8a31bf3bc5f15b95a98b59ae3db54fe67491608b) would you be up to adding those changes here as well?

**Comment by huyikai** [comment]
> Heads up we got a [few more updates on this page](https://github.com/withastro/docs/commit/8a31bf3bc5f15b95a98b59ae3db54fe67491608b) would you be up to adding those changes here as well?

I've added these updates.

File: src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx [hunk 1]
```diff
@@ -255,45 +348,54 @@ export default defineMarkdocConfig({
...
     },
   },
 });
 ```
 
-### 自定义 Markdoc 节点 / 元素
+<ReadMore>查看 [Markdoc 节点文档](https://markdoc.dev/docs/nodes#built-in-nodes) 了解所有内置节点及其属性。</ReadMore>
 
-你可能想要将标准的 Markdown 元素，例如段落和加粗文本，渲染为 Astro 组件。为此，你可以配置一个 [Markdoc 节点][markdoc-nodes]。如果给定的节点接收到属性，它们可将作为组件 props 来使用。
+### Custom headings
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx:372]: 这里需要翻译吧

File: src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx [hunk 2]
```diff
@@ -471,89 +542,23 @@ export default defineMarkdocConfig({
...
 ```js title="markdoc.config.mjs"
-import { defineMarkdocConfig } from '@astrojs/markdoc/config';
+import { defineMarkdocConfig, nodes } from '@astrojs/markdoc/config';
 
 export default defineMarkdocConfig({
-  variables: {
-    environment: process.env.IS_PROD ? 'prod' : 'dev',
+  nodes: {
+    document: {
+      ...nodes.document, // Apply defaults for other options
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx:555]: 注释翻译一下

File: src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx [hunk 3]
```diff
@@ -255,45 +348,54 @@ export default defineMarkdocConfig({
...
 ```js title="markdoc.config.mjs"
 import { defineMarkdocConfig, nodes, component } from '@astrojs/markdoc/config';
 
 export default defineMarkdocConfig({
   nodes: {
-    blockquote: {
-      ...nodes.blockquote, // 应用默认的其他配置项
-      render: component('./src/components/Quote.astro'),
+    heading: {
+      ...nodes.heading, // Preserve default anchor link generation
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx:384]: 这里的注释

**Comment by liruifengv** [comment]
image

最后这一部分也翻译一下吧

**Comment by huyikai** [comment]
> image
> 
> 最后这一部分也翻译一下吧

这个 follow 这么翻译合适吗

File: src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx [hunk 4]
```diff
@@ -597,20 +602,19 @@ Astro Markdoc 集成处理了在 `markdoc.config.js` 文件中不可用的 Markd
...
+{% custom-tag %} ### 标签可以缩进以提高可读性
 
     {% another-custom-tag %}
-      This is easier to follow when there is a lot of nesting
+      当嵌套很多时，这样更容易跟随
     {% /another-custom-tag %}
 
 {% /custom-tag %}
 {% /custom-tag %}
-```
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx:613]: 最后样式出问题了

File: src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx [hunk 5]
```diff
@@ -597,20 +602,19 @@ Astro Markdoc 集成处理了在 `markdoc.config.js` 文件中不可用的 Markd
...
-# Note: Can use either spaces or tabs for indentation
+# 注意：可以使用空格或制表符进行缩进
 
 {% custom-tag %}
-{% custom-tag %} ### Tags can be indented for better readability
+{% custom-tag %} ### 标签可以缩进以提高可读性
 
     {% another-custom-tag %}
-      This is easier to follow when there is a lot of nesting
+      当嵌套很多时，这样更容易跟随
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/markdoc.mdx:613]:
```suggestion
      当有大量嵌套时，这一点很容易遵循。
```

**Review by liruifengv — APPROVED** [review]
LGTM

**Pull request merged**